alf/drivers/off_policy_driver.py

```diff
@@ -153,14 +201,62 @@ def transpose2(x, dim1, dim2):
 
         return tf.nest.map_structure(lambda x: transpose2(x, 0, 1), nest)
 
-    def train(self, experience: Experience):
```

Comment:
why not keep previous implementation  ,  sample mini_batch with replay buffer and perform update  

as following pseudo code do

	for _ in range(num_iterations):
		collect new data and store to replay buffer
		for _ in range(num_updates):
		    experience = sample mini batch from replay buffer
		    driver.train(experience)

		if clear_replay_buffer:
		   replay_buffer.clear()

Reply:
In the case of PPO, it wants to go over all the data exact num_update times. Randomly sample from replay buffer cannot achieve this.